apps/web/src/components/v2Editor/customBlocks/visualization/index.tsx

```diff
@@ -202,6 +203,34 @@ function VisualizationBlock(props: Props) {
     props.block.setAttribute('controlsHidden', !controlsHidden)
   }, [controlsHidden, props.block])
 
+  const onExportToPNG = async () => {
+    // we don't need to check if props.renderer is undefined because the application sets as 'canvas' in this case
+    if (
+      props.renderer === 'svg' ||
+      chartType === 'number' ||
+      chartType === 'trend'
+    )
+      return
+
+    // if the controls are visible the canvas got shrinked, making the export smaller
```

Comment:
This is fine! I'm okay with that! Great idea here.

Minor grammatical nit:

```suggestion
    // if the controls are visible the canvas shrinks, making the export smaller
```